jax/nn/__init__.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Common neural network activations and other functions."""
```

Comment:
Can we not put code in the `__init__.py` file? That's another thing we've avoided so far in the project. Instead, just put this code in a regular module and the import it in the `__init__.py` file. Could go with calling this module util.py, i.e. jax.nn.util, but any name is fine.